### ISSUE TITLE
If you want to use `GithubLink` component, you must specify: `owner`, `repo` and `branch`.

### ISSUE THREAD

**Issue by productdevbook** [nuxt-themes/docus#797]
https://opensource.huntersofbook.com/global/i18n

refresh page and see

github: https://github.com/huntersofbook/huntersofbook/tree/main/docs

![image](https://user-images.githubusercontent.com/38668796/215325870-b14a2034-6e35-4b86-8530-18b6f8407db8.png)

**Comment by esoubiran-aneo** [comment]
Hello, what is the issue ?

**Comment by esoubiran-aneo** [comment]
Please see configuration docs https://docus.dev/introduction/configuration

**Comment by productdevbook** [comment]
@esoubiran-aneo The codes are already open with the settings. I shared the repo above. I wrote what the problem was. This error pops up when I refresh the page.

**Comment by Barbapapazes** [comment]
Yes, you need to update this object: https://github.com/huntersofbook/huntersofbook/blob/main/docs/app.config.ts#L26

Using this part of the table

![image](https://user-images.githubusercontent.com/45267552/215475912-7a205b48-5d56-4d93-90ae-edf8f21c947d.png)


Exemple: https://github.com/aneoconsulting/armonik-docs-theme/blob/main/app.config.ts#L23
